GOVERNANCE.md

```diff
@@ -1,3 +1,246 @@
+Seats on the Steering Committee are held by an individual, not by their
+employer.
+
+The current membership of the committee is currently (listed alphabetically by
+first name):
+
+| &nbsp;                                                         | Member           | Organization |
+| -------------------------------------------------------------- | ---------------- | ------------ |
+
+[TODO: Pending elections]
+
+## Decision process
+
+The Steering Committee desires to always reach consensus.
+
+Decisions requiring a vote include: issuing written policy, amending existing
+written policy, creating, removing, or modifying a committee or SIG, all spending,
+hiring, and contracting, official responses to publicly raised issues, or any
+other decisions that at least two of the members present decide require a vote.
+
+Decisions are made in meetings when a quorum of more than half of the members
+are present (and all members have been informed of the meeting), and may pass
+with more than half the members of the committee supporting it.
+
+## Getting in touch
+
+Open an issue on [the governance repo](https://github.com/Kuadrant/governance) and indicate that you would like
+attention from the steering committee.
+
+## Composition
+
+The steering committee has 3 seats. These seats are
+open to any project contributor. See
+[candidate eligibility](#candidate-eligibility) for a definition.
+
+## Election Procedure
+
+### Timeline
+
+Steering Committee elections are held annually. Six weeks or more before the
+election, the Steering Committee will appoint Election Officer(s) (see below).
+Four weeks or more before the election, the Election Officer(s) will issue a
+call for nominations, publish the list of voters, and open the call for
+exceptions. Three days before the election the call for nominations and exceptions
+will be closed. The election will be open for voting not less than two weeks and
+not more than four. The results of the election will be announced within one
+week of closing the election. New Steering Committee members will take office on
+October 1st of each year.
+
+### Election Officer(s)
+
+Six weeks or more before the election, the Steering Committee will appoint
+between one and three Election Officer(s) to administer the election. Elections
+Officers will be community members in good standing who are eligible to
+vote, are not running for Steering in that election, who are not currently part
+of the Steering Committee and can make a public promise of impartiality. They
+will be responsible for:
+
+- Making all announcements associated with the election
+- Preparing and distributing electronic ballots
+- Judging exception requests
+- Assisting candidates in preparing and sharing statements
+- Tallying voting results according to the rules in this charter
+
+### Eligibility to Vote
+
+Anyone who has at least 10 contributions in the last 12 months is
+eligible to vote in the Steering election. Contributions are defined as opening PRs,
```

Comment:
So somewhat rules out "users", is this on purpose? Desirable?

Reply:
I'm not against users having a say.
Is it feasible to keep a record of users though.
How would you see that playing out for voting.

Reply:
Given we have a solution that probably mostly targets developers (even if YAML ones), would it be crazy to require users wanting to vote to "record" as users of any of our components? Or just "star" one of the repo? I guess the problem is needing a way to track individual votes... so a platform that would let users link to their GH account, and that's it? Thinking out loud here, I might well be missing a big part of the problem to be solved.

Again, given the responsibilities of steering committee members being both technical but also "community" ones, being more inclusive in the way they are elected seems desirable... And I feel having "only contributors", given the criteria above, being the voters, seems a little like a close feedback loop to me. But again, maybe I'm missing some aspect of this and could also well be a little too much of a dreamer 😜

Reply:
I don't know any project that allows "users" to vote, simply because there isn't any good way to verify a user's eligibility.  The closest I've seen is that some projects, like Cert-Manager, allow representatives of the organizations in their Adopters.md file to vote on an end user seat.